.changeset/actionlist2-actionmenu2-prefix-types.md

```diff
@@ -0,0 +1,17 @@
+---
+'@primer/react': major
+---
+
+Prefix ActionList for exported types from ActionList2
+
+ListProps → ActionListProps
+GroupProps → ActionListGroupProps
+ItemProps → ActionListItemProps
+DescriptionProps → ActionListDescriptionProps
+LeadingVisualProps → ActionListLeadingVisualProps,
+TrailingVisualProps → ActionListTrailingVisualProps
+
+Prefix ActionMenu for exported types from ActionMenu2
+
+MenuButtonProps → ActionMenuButtonProps
+MenuAnchorProps → ActionMenuAnchorProps
```

Comment:
This doesn't format well in markdown: 
<img width="772" alt="CleanShot 2022-02-23 at 08 06 02@2x" src="https://user-images.githubusercontent.com/4608155/155358486-810986f3-dfee-41bd-82b1-7e5789b68dc8.png">

I think this formatting might work better:
````suggestion
`ActionList2` exported types are now prefixed with `ActionList`:

```
ListProps → ActionListProps
GroupProps → ActionListGroupProps
ItemProps → ActionListItemProps
DescriptionProps → ActionListDescriptionProps
LeadingVisualProps → ActionListLeadingVisualProps,
TrailingVisualProps → ActionListTrailingVisualProps
```

`ActionMenu2` exported types are now prefixed with `ActionMenu`:

```
MenuButtonProps → ActionMenuButtonProps
MenuAnchorProps → ActionMenuAnchorProps
```
````